enforcer/pom.xml

```diff
@@ -387,6 +385,11 @@
             <artifactId>Java-WebSocket</artifactId>
             <version>1.5.1</version>
         </dependency>
-
+        <!-- slf4j binding -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>2.13.3</version>
```

Comment:
Lets move the version to root pom

Reply:
my bad.
Will fix

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/2021/commits/8ef5c052d366170ddbee991a9d2b752ee291e294